common/client.go

```diff
@@ -66,6 +67,20 @@ func (c *DatabricksClient) WorkspaceClient() (*databricks.WorkspaceClient, error
 	return w, nil
 }
 
+func (c *DatabricksClient) AccountClient() (*databricks.AccountClient, error) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	if c.cachedAccountClient != nil {
+		return c.cachedAccountClient, nil
+	}
+	acc, err := databricks.NewAccountClient((*databricks.Config)(c.DatabricksClient.Config))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
AccountClient has AccountUsersService instead of currentUsersService that's why we cannot override implementation with cachedMe similar to account. 

This would require another type:

```
type cachedMeAccount struct {
	internalImpl iam.AccountUsersService
	cachedUser   *iam.User
	mu           sync.Mutex
}

func (a *cachedMeAccount) Me(ctx context.Context) (*iam.User, error) {
```

But we cannot do it because AccountUsersService requires other methods to be implemented such as Create